packages/test/end-to-end-tests/src/test/loaderTest.spec.ts

```diff
@@ -33,6 +35,19 @@ class TestSharedDataObject1 extends DataObject {
     public get _context() {
         return this.context;
     }
+
+    // this is for returning query params in the last test
```

Comment:
The last test will change if someone adds more tests to this file :-). Maybe say something like "This will return back query paras (if any) in the request. This is used in tests that verify query params work correctly with loader.request".